SwitchSpeak-iOS/SwitchSpeak-iOS/CrumbManager.swift

```diff
@@ -54,8 +62,13 @@ class CrumbStack {
     // The stack holding the breadcrumbs.
     private var items: [Crumb]
     
+    private var crumbWidth: CGFloat
+    
     init() {
         self.items = []
+        let screenSize = UIScreen.main.bounds
+        let screenWidth = screenSize.width
+        self.crumbWidth = 0.2*screenWidth
```

Comment:
Please put spaces around all operators:
`0.2 * screenWidth`

Reply:
Also, please put the 0.2 in a private static variable. [See here](https://developer.apple.com/library/content/documentation/Swift/Conceptual/Swift_Programming_Language/Properties.html#//apple_ref/doc/uid/TP40014097-CH14-ID264) for documentation on how to do that.